miscellaneous/csv.md

```diff
@@ -0,0 +1,195 @@
+# CSV
+
+## Introduction
+
+Phlex also has a CSV table builder interface that should make you feel right at home if you're already comfortable working within Phlex. It carries over Phlex's design principles of safety, speed, and streamability.
```

Comment:
I think we should talk specifically about streaming from Rails somewhere in this guide.